ReactWindows/ReactNative/UIManager/UIImplementation.cs

```diff
@@ -659,6 +660,9 @@ private void HandleCreateView(ReactShadowNode cssNode, int rootViewTag, ReactSty
         private ReactShadowNode CreateRootShadowNode()
         {
             var rootCssNode = new ReactShadowNode();
+            if (InternationalizationUtil.IsRightToLeft)
+                rootCssNode.Direction = Facebook.CSSLayout.CSSDirection.RTL;
```

Comment:
Have you confirmed if the CSSDirection is inherited? We use different binaries than are used for iOS and Android.